plugins/socks5-proxy.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: socks5-proxy
+spec:
+  version: "v1.0.0"
+  homepage: https://github.com/yokawasa/kubectl-plugin-socks5-proxy
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/yokawasa/kubectl-plugin-socks5-proxy/archive/v1.0.0.zip
+    sha256: b8d96ab7b7d2ba23d973eeb3a22080829073ae79f8cad5966740fc1ceaf766d3
+    files:
+    - from: "./*/kubectl-socks5-proxy"
+      to: .
+    - from: "./*/LICENSE"
+      to: .
+    bin: "./kubectl-socks5-proxy"
+  shortDescription: A kubectl plugin that creates a local SOCKS5 proxy through which you can access to Services or Pods in a Kubernetes cluster.
+  description: |
+    A kubectl plugin that creates a local SOCKS5 proxy through which you can
+    access to Services or Pods in a Kubernetes cluster.
+    
+    What the plugin actually does is that it create a SOCKS proxy server Pod 
+    in a Kubernetes cluster and forwards a local port (default:1080) to the proxy.
```

Comment:
Is there also a command to tear down this proxy Pod? You don't need to mention it here, but it would be nice if the respective command would be shown after starting the proxy successfully.

Reply:
Thank you for the suggestion! I added message on cleaning up the proxy Pod -https://github.com/yokawasa/kubectl-plugin-socks5-proxy/blob/main/kubectl-socks5-proxy#L81-L82